README.md

```diff
@@ -58,36 +63,51 @@ ansible_ssh_pass=vargrant
 ansible_private_key_file=./vm-testing/images/rhel9-vm/.vagrant/machines/trustification/libvirt/private_key
 ```
 
-2. Create  `ansible.cfg` file  in the project with a single VM in the `trustification` group:
+2. Create `ansible.cfg` file in the project with a single VM in the `trustification` group:
+
 ```
 [defaults]
 inventory = ./inventory.ini
 host_key_checking = False
 ```
-3. Add your RH credentials, choose one:
- - as a env variables `SUBSCRIPTION_USERNAME` and `SUBSCRIPTION_PASSWORD` 
- - in the Vagrant file `config.registration.username` and `config.registration.password`
- - during the vagrant up in the registration prompt
 
+3. Add the subscription, registry and certificates information :
+
+- For Red Hat subscription define :
+  export TPA_SINGLE_NODE_REGISTRATION_USERNAME=<Your Red Hat subscription username>
```

Comment:
Are the same the credentials for subscription and tò access the registry ?

Reply:
As I explained above, they could be the same but when using auth access token they are different.